pkg/networkservice/common/heal/eventloop.go

```diff
@@ -32,18 +32,21 @@ import (
 	"github.com/networkservicemesh/sdk/pkg/tools/log"
 )
 
-const attemptAfter = time.Millisecond * 200
+const (
+	attemptAfter = time.Millisecond * 200
```

Comment:
attemptAfter is not actually using as we realised with @glazychev-art 

Please remove this and all dependent code.